### DISCUSSION TITLE
Missing ISO 7589 Illuminant?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1166] [Q&A] [answered]
'ISO 7589 Sensitometric Daylight'
'ISO 7589 Sensitometric Studio Tungsten'
'ISO 7589 Sensitometric Photoflood'
'ISO 7589 Sensitometric Printer'

'ISO 7589 Photographic Daylight'
'ISO 7589 Studio Tungsten'
'ISO 7589 Photoflood'

It seems that the light source for 'Printer' used by the sensitometric specification has not been included.  Purposeful?  Or oversight?

ISO 7589:2002, pg. 9:

> "**A.5 Printer source**
> An incandescent tungsten or tungsten-halogen lamp operating at a distribution temperature of 2 856 K in combination with a Schott GG395 glass filter (or equivalent) of appropriate thickness may be used. Table A.2 illustrates how the spectral power distribution is determined and shows how the SDI is calculated to determine whether it meets the specification for ISO sensitometric printer illuminant."

- J

**Comment by thomasmansencal** [comment, marked as answer]
Hey @blue-j,

Those are informative examples in Annex A, the ISO illuminants are the ones you quoted above.

![image](https://github.com/colour-science/colour/assets/99779/0b5bd38d-c7a6-4c80-a3ac-1451f9055867)

Cheers,

Thomas
